gulpfile.coffee

```diff
@@ -79,6 +87,32 @@ gulp.task "watcher", ->
 		.pipe(gulpWebpack(config))
 		.pipe(gulp.dest("build/"))
 
+gulp.task "version", (callback) ->
+
+	async.series [
+		(cb) -> command("git rev-parse --abbrev-ref HEAD", cb) # branch
+		(cb) -> command("git describe --always --dirty", cb) # hash
+		(cb) -> command("git rev-list --count HEAD", cb) # build
+	], (err, results) ->
+
+		info = 
+			branch: results[0]
+			hash: results[1]
+			build: results[2]
+			date: Math.floor(Date.now() / 1000)
+
+		console.log "version:#{info.branch}/#{info.hash} build:#{info.build}"
+		 
+		task = gulp.src("framer/Version.coffee.template")
+			.pipe(template(info))
+			.pipe(rename({
+				basename: "Version",
+				extname: ".coffee"
+			}))
+			.pipe(gulp.dest("framer"))
+
+		callback(null, task)
+
```

Comment:
Yes because things get messed up when I insert anything after the webpack build, I just generate a file with version information before. I don't get the last sentence?